openstudiocore/resources/model/OpenStudio.idd

```diff
@@ -15708,6 +15710,47 @@ OS:Controller:MechanicalVentilation,
        \type object-list
        \object-list DesignSpecificationZoneAirDistributionNames
 
+OS:ZoneControl:ContaminantController,
+       \min-fields 8
+  A1, \field Handle
+       \type handle
+       \required-field
+  A2 , \field Name
+       \required-field
+  A3 , \field Controlled Zone Name
```

Comment:
This is one of those data model mismatch things, it's possible for multiple ZoneControl:ContaminantControllers to reference the same zone, this just has to get enforced in the API I guess